mbf_abstract_nav/src/abstract_controller_execution.cpp

```diff
@@ -411,8 +411,8 @@ void AbstractControllerExecution::run()
           else
           {
             setState(NO_LOCAL_CMD); // useful for server feedback
-            // we keep on moving if we have retries left or if the user has granted us some patience.
-            moving_ = max_retries_ || !patience_.isZero();
+            // keep trying if we have non-zero retries left
+            moving_ = max_retries_;
```

Comment:
so this will be casted to zero only if max_retries is zero and we will continue moving if max_retries is disabled (aka. negative)? I like!

Reply:
maybe add a comment that -1 is also casted to true (I had to double check that) or write `moving_ = max_retries != 0` so we express it clearer for people like me;)

Reply:
well,,,   -1 is also non-zero

Reply:
Could you explain why we should remove `!patience_.isZero();`?

Reply:
Before:  we keep retrying if retries is > 0 or we have not exhausted a *non-zero* patience
After:     we keep retrying if retries is > 0

That is, if retries is 0, we never retry, regardless of patience. That's what we have documented on the wiki